src/Mobs/AggressiveMonster.h

```diff
@@ -20,7 +20,7 @@ class cAggressiveMonster :
 	virtual void InStateChasing(std::chrono::milliseconds a_Dt) override;
 	
 	virtual void EventSeePlayer(cEntity *) override;
-	virtual void Attack(std::chrono::milliseconds a_Dt);
+	virtual bool Attack(std::chrono::milliseconds a_Dt);
```

Comment:
This could use some commenting - when it's called, what it's supposed to do, what the default implementation does and what the return value means.